src/components/Pressable/PressableWithFeedback.js

```diff
@@ -39,25 +39,22 @@ const PressableWithFeedbackDefaultProps = {
 
 const PressableWithFeedback = forwardRef((props, ref) => {
     const propsWithoutWrapperStyles = _.omit(props, omittedProps);
-    const [disabled, setDisabled] = useState(props.disabled);
-    const [isPressed, setIsPressed] = useState(false);
+    const [isExecuting, setExecuting] = useState(false);
+    const [isPressed, setPressed] = useState(false);
```

Comment:
Revert this. It was isPressed, setIsPressed

Reply:
Updated 👍